### PR TITLE
Test/mobile sdk split key

### PR DESCRIPTION
**Pull request by Ash-KODES**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/0chain/errors v1.0.3
 	github.com/0chain/gosdk v1.11.1-0.20240204123736-4997432ef46e
 	github.com/go-resty/resty/v2 v2.7.0
-	github.com/herumi/bls-go-binary v1.31.0
+	github.com/herumi/bls-go-binary v1.33.0
```
Comment (Jayashsatolia403) [inline go.mod:9]: Why do you need to update bls version?

File: tests/api_tests/split_key_wallet_mobile_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		// this represents number of split keys made from private key
+		numSplit := 2
+		signatureScheme := "bls0chain"
+		wStr, err := sdk.SplitKeys(stringPrivateKey, signatureScheme, numSplit)
+		if err != nil {
+			fmt.Println("Error while spliting keys:", err)
+			return
+		}
+		/*
+			{
```
Comment (Jayashsatolia403) [inline tests/api_tests/split_key_wallet_mobile_test.go:32]: Remove comments.

**Review by Jayashsatolia403 — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (Ash-KODES) [reply go.mod]: Actually don't need it,degraded that.

**Review by Jayashsatolia403 — APPROVED** [review]
lgtm, merge on green run

**Pull request merged**